src/frontend/navigator.cpp

```diff
@@ -206,6 +206,23 @@ void Navigator::requestScreen(int requestedScreen,
   logger.debug() << "Unable to show the requested screen";
 }
 
+void Navigator::requestDeepLink(const QUrl& url) {
+  logger.debug() << "Recevied nav link:" << url.toString();
+
+  // Quick and dirty navigation handler.
+  // Lookup the screen by name using QMetaEnum
+  QString topPath = url.path().section('/', 0, 0, QString::SectionSkipEmpty);
+  QString name = QString("Screen%1").arg(topPath);
+  const QMetaEnum meta = QMetaEnum::fromType<MozillaVPN::CustomScreen>();
+  for (int index = 0; meta.keyCount(); index++) {
```

Comment:
Nice! Consider adding a comment that this is enumerating over all the screen names in the app, by using the CustomScreen enum type's value strings.

Reply:
Done